README.md

```diff
@@ -197,6 +201,75 @@ NDArray succesfully created in btune_config.b2nd
 
 Here we set the tradeoff to 0.3 and the performance mode to `DECOMP`.
 
+## Btune quality mode
+
+Now Btune can work not only taking into account
+the compression ratio and speed, but also the quality. This is intended
+to work with datasets made out of images which datatypes are integers. Another 
+requirement is that the `blosc2_grok` plugin will have to be installed in the
+system.
+
+To use it, you have to set the `BTUNE_TRADEOFF` to a tuple of 3 values 
+`(cratio, speed, quality)`. These must sum up to 1. The bigger one value is,
+the more important it will be. 
+
+In the `examples` directory there is a Python example `lossy.py` which creates a NDArray
+made out of 10 chunks (one chunk per image). You can run it with:
+
+```shell
+BTUNE_TRACE=1 python lossy.py MI04_020751.tif
+
+-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+Btune version: 1.1.2
+Performance Mode: COMP, Compression tradeoff: (0.500000, 0.300000, 0.200000), Bandwidth: 20 GB/s
+Behaviour: Waits - 0, Softs - 0, Hards - 0, Repeat Mode - (null)
+Behaviour: Waits - 0, Softs - 5, Hards - 10, Repeat Mode - STOP
+INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
+TRACE: time load model: 0.000281
+|    Codec   | Filter | Split | C.Level | C.Threads | D.Threads |  S.Score  |  C.Ratio   |   Btune State   | Readapt | Winner
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0137 |      8.01x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0374 |      8.01x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0387 |      8.01x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0362 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0372 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0384 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0389 |      8.01x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0371 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0361 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0377 |      8.01x |    CODEC_FILTER |    HARD | -
+```
+The tradeoff used tells Btune that you care a lot about compression ratio but not as much
+about speed or the quality loss. According to that, Btune uses the `blosc2_grok` codec 
```

Comment:
Btune uses -> Btune predicts

Reply:
Fixed in 9fcfb27cc73198307be5bb26ef401e393db473a5.